### DISCUSSION TITLE
Milkdrop / Spheck implementation

### DISCUSSION THREAD

**Discussion by SamAresOfficial** [TT-ReBORN/Georgia-ReBORN#196] [Ideas]
I would love if a visualization tab was added beside rating and simply displayed milkdrop visualizer between the player at the bottom and the top bar if its possible.

**Comment by TT-ReBORN** [comment]
Hi @SamAresOfficial,

please visit and read this thread here: https://github.com/TT-ReBORN/Georgia-ReBORN/discussions/182

It means that you can not directly implement it, only with embedding it into an external floating panel ( window ).

-TT
